### ISSUE TITLE
Upgrade to bootstrap 4

### ISSUE THREAD

**Issue by HarshKhandeparkar** [publiclab/image-sequencer#938]
### Please describe the problem (or idea)
We currently use bootstrap 3.7. Bootstrap 4 has some significant improvements/changes. Let's upgrade to v4.

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by jywarren** [comment]
big change! But yeah, we can -- our HTML is relatively minimal. +1 after we
get the Bootstrap pane UI merged!

**Comment by Harshithpabbati** [comment]
I will do this :smile:

**Comment by Harshithpabbati** [comment]
Should I do it or leave to some new comers they may do it ???

**Comment by HarshKhandeparkar** [comment]
I think you should do this as this is a relatively difficult task. What do you say?

**Comment by Harshithpabbati** [comment]
Ok fine then I will do this!!

**Comment by ananya** [comment]
Can I work on this?

**Comment by Harshithpabbati** [comment]
Fine then you can.

**Comment by Harshithpabbati** [comment]
But you need to wait for some time. Let the panel be merged and then you can start your work !!!

**Comment by ananya** [comment]
Thank you :smiley: Let me know once I can start working

**Comment by Harshithpabbati** [comment]
@ananya you can start working with it :tada:

**Comment by ananya** [comment]
@HarshKhandeparkar yeah okay :smiley: I will start working on this in the coming week.

**Comment by Harshithpabbati** [comment]
@ananya check the name 😂

**Comment by ananya** [comment]
haha fixed :stuck_out_tongue:

**Comment by ananya** [comment]
![Screenshot from 2019-04-02 21-22-23](https://user-images.githubusercontent.com/33188930/55416993-73057d00-558d-11e9-8822-2fc78757c615.png)
![Screenshot from 2019-04-02 21-22-28](https://user-images.githubusercontent.com/33188930/55416994-73057d00-558d-11e9-8e1d-f5d4f413ac51.png)

My localhost is looking like this on installing the project. Is it fine or I am missing something?

**Comment by Harshithpabbati** [comment]
No. It looks like this.
![image-se](https://user-images.githubusercontent.com/43822585/55417181-cd9ed900-558d-11e9-9b0d-c8589229e84f.png)

**Comment by Harshithpabbati** [comment]
Your branch is not upto date. Pull the changes

**Comment by ananya** [comment]
My branch is upto date. I just have to "npm run setup" and "npm start" sfter cloning the project right?

**Comment by HarshKhandeparkar** [comment]
I think you have cloned jywarren/image-sequencer. Clone publiclab/image-sequencer

**Comment by ananya** [comment]
![Screenshot from 2019-04-02 22-05-52](https://user-images.githubusercontent.com/33188930/55419953-84518800-5593-11e9-85bd-ca3aad9a0109.png)

@HarshKhandeparkar its publiclab/image-sequencer only

**Comment by jywarren** [comment]
Could it be a caching issue? Thanks for working together to figure this out!

**Comment by HarshKhandeparkar** [comment]
@ananya try `git fetch upstream` first.

**Comment by HarshKhandeparkar** [comment]
You can also run
```
git fetch upstream
git reset --hard upstream/main
```

Bur be careful as this will overwrite any existing changes

**Comment by jywarren** [comment]
But I am still seeing intermittent saved caches when testing things... so
wondering if it could be an issue!

**Comment by HarshKhandeparkar** [comment]
@ananya what browser ro you use?

**Comment by ananya** [comment]
@Harshithpabbati i am on chrome

**Comment by HarshKhandeparkar** [comment]
Is that chrome or chromium? Try upgrading it to the latest version.

**Comment by HarshKhandeparkar** [comment]
Is it tye latest version?

**Comment by ananya** [comment]
it is chrome. And I tried to fetch also but it is still showing the same page.

**Comment by HarshKhandeparkar** [comment]
Also can you run `ls` in a terminal and provide a screenshot of the output?

**Comment by HarshKhandeparkar** [comment]
Btw, did you try clearing the cache?

**Comment by ananya** [comment]
It worked somehow :smiley: . Though this was the result i got when i installed the project for the first time, then how could this be a cache problem :thinking:

**Comment by HarshKhandeparkar** [comment]
Maybe it cached once when your project was not up to date? Who knows but anyways I'm that you managed to fix it👍. Feel free to ask any other queries as well. 😊

**Comment by ananya** [comment]
Oh Okay. Thank you for the help :smiley:
